content/articles/redirector.markdown

```diff
@@ -13,17 +13,16 @@ categories:
 {:toc}
 
 ---
+<info>
+HTTP redirects are available to all plans. HTTPS redirects are available only on the following [plans](https://dnsimple.com/pricing): Professional, Business, Expert Reseller or Master Reseller.
```

Comment:
On my first readthrough, I misread this paragraph. Does it make sense to anyone else to bold or otherwise call attention to "HTTP" and "HTTPS" to emphasize the difference? 
>**HTTP** redirects are available to all plans. **HTTPS** redirects are available only on the following [plans](https://dnsimple.com/pricing): Professional, Business, Expert Reseller or Master Reseller.

Reply:
I'm here for bolding 👍